web_src/css/repo.css

```diff
@@ -123,6 +123,12 @@ td .commit-summary {
   gap: 0.25em;
 }
 
+@media (max-width: 767.98px) {
```

Comment:
this removed in https://github.com/go-gitea/gitea/commit/079a1ffe8f38e0c7570079b21a4d8cec4d90e923#diff-4336f7c2a837a4d5b935189073546814a6d1b6ca1390e41dd21c1c6d1366aadfL126

will we add again?

Reply:
Yup, need to add it again ....

The commit ID classes was changed from "ui sha label" to "ui label commit-id-short", the removal is not right.